### DISCUSSION TITLE
API is Down

### DISCUSSION THREAD

**Discussion by charlessnow** [lukePeavey/quotable#132] [Archived]
**Describe the bug**
Some error occurred that made the api unusable.
**To Reproduce**
1. I Made the following API request

```HTTP
GET https://api.quotable.io/random?maxLength=50
```

**Expected Response**
The api request result is as follows:
![image](https://user-images.githubusercontent.com/15224389/174251430-79eb0efd-6955-476c-b154-88d94faad5e6.png)
Please check the situation,Thank you.

**Comment by egoTr** [comment]
I got the same error. 😥

**Comment by khushalpatel1** [comment]
same error

**Comment by lukePeavey** [comment]
**I am looking into this now.** I will provide an update as soon as I known more.

**Comment by lukePeavey** [comment]
fixed
